scripts/test.backend.sh

```diff
@@ -4,7 +4,8 @@ POCKET_IC_SERVER_VERSION=6.0.0
 OISY_UPGRADE_VERSIONS="v0.0.13,v0.0.19"
```

Comment:
The PR is really quite big.  Normally we try to keep PRs to under 5 files and maybe 200 lines of code.  I would recommend putting the changes to `test.backend.sh` in a separate PR, for example.